interactive_ai/Makefile

```diff
@@ -16,9 +16,10 @@ PY_SUBPROJECTS := $(call find_subprojects,.,pyproject.toml)
 SUBPROJECTS := $(GO_SUBPROJECTS) $(PY_SUBPROJECTS)
 
 WORKFLOWS := $(call find_subprojects,workflows,job,3)
+TRAINERS := $(call find_subprojects,workflows,.python,4)  # needed for otx2 images
```

Comment:
The search filter doesn't seem very robust. If for any reason we add a `.python` file in any other workflows, then this filter will mistakenly include that workflow. I suggest to restrict the search to `workflows/train/trainer`, and maybe use the `Makefile` instead of `.python` which is a more reliable marker of a buildable component.